runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -26,59 +26,56 @@ namespace misc {
     ssize_t b;
 
     Interval();
-    explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
```

Comment:
Happy to put it back. However: I don't understand what you mean about `size_t` and `ssize_t` conversions. My understanding of the `explicit` keyword is that it will prevent:

    IntervalSize s = { 1, 2 };

from compiling, and require:

    IntervalSize s(1, 2);

instead.